cmd/tpmk/sshclient.go

```diff
@@ -194,12 +197,29 @@ func runSSHClient(opt sshClientOptions, args []string) error {
 		HostKeyCallback: hostKeyCallback,
 	}
 
-	// Connect to the SSH server and start a session
-	client, err := ssh.Dial("tcp", host, config)
-	if err != nil {
-		return errors.Wrap(err, "connecting to "+host)
+	var client Client
```

Comment:
Are you sure this is the final version you were able to build successfully? This should probably be `*ssh.Client`.